java/java.lsp.server/vscode/BUILD.md

```diff
@@ -35,10 +42,9 @@ To build the VS Code extension invoke:
 ```bash
 netbeans$ ant build
 netbeans$ cd java/java.lsp.server
-java.lsp.server$ ant build-vscode-ext -D3rdparty.modules=.*nbjavac.*
+java.lsp.server$ ant build-vscode-ext -Dvsix.version=x.y.z
 ```
-
-The `3rdparty.modules` property doesn't have to be set at all.
```

Comment:
Either the `3rdparty.modules` property remain to be documented, or it shall be removed. If it doesn't work with `m2://` protocol (which it didn't), that should be fixed.